internal/common/mqtt/mqttconfig.go

```diff
@@ -0,0 +1,82 @@
+package mqtt
```

Comment:
Need a copyright and license statement at the head of file here.

Reply:
Done!